frontend/src/domain/intro/components/Footer/index.js

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 import styled from 'styled-components';
 import { Link } from 'react-router-dom';
-import GrayLogo from '../../../../assets/images/waktaverse_word_logo_gray.svg';
-import DarkGrayLogo from '../../../../assets/images/waktaverse_word_logo_darkgray.svg';
+import GrayLogo from '../../../../assets/images/wakgames_logo/waktaverse_word_logo_gray.svg';
+import DarkGrayLogo from '../../../../assets/images/wakgames_logo/waktaverse_word_logo_darkgray.svg';
```

Comment:
```suggestion
import GrayLogo from 'assets/images/wakgames_logo/waktaverse_word_logo_gray.svg';
import DarkGrayLogo from 'assets/images/wakgames_logo/waktaverse_word_logo_darkgray.svg';
```
아마 기본 폴더가 frontend 부터일거라서 앞에 ../ 없어도 될거에요!